src/Containers/Profile/Profile.scss

```diff
@@ -0,0 +1,194 @@
+@import '../../assets/sass/global.scss';
+#profile {
+  input,
+  button {
+    width: 80%;
+  }
+  button {
+    width: 80%;
+  }
+  label {
+    color: #444;
+  }
+  textarea {
+    border: 2px solid #efefef;
+    margin-bottom: 10px;
+    padding: 0 6px;
+    width: 90%;
+    background: #fafafa;
+    &:focus {
+      outline: none !important;
+      border: 2px solid rgba(#aca9a9, 0.5);
+      box-shadow: 2px 0 6px rgba(#efefef, 0.8);
+    }
+  }
+  .navbar {
+    background-color: white;
+  }
+}
+
+.body {
+  padding-left: 14%;
+
+  div,
```

Comment:
I can see here you are giving a style to every div on every page can you wrap it inside of `#profile`